### PR TITLE
Update design to Ember-AppUniversum and more

### PR DESCRIPTION
**Pull request by benjay10**
This PR includes:

*   Complete redesign using (Ember-)AppUniversum, including the addition of alerts, pills, lists, toolbars, ...
*   Using imports from AppUniversum to make little tweaks (such as column widths, collapsing borders, ...)
*   Using Ember-Resources (I still don't like it very much :p) on top of ember-concurrency tasks
*   Tuning for least amount of queries (using resources and a form of memoisation)
*   More use of Ember components to reduce duplication in templates
*   Proper translations using Ember-Intl
*   Input triggering on {{on "input" ...}} instead of "keyup" for more expected behaviour
*   Increased debounce timeout to 500 ms for more comfortable typing and less fetches
*   Introducing memoisation on the decision and articles to drastically speed up repeated scrolling through pages and context switches
*   Cancellation of requests when closing modal, closing card, or navigating away from article search results using AbortController and signal on the fetch()

Known issues:

*   Sometimes, inserting a reference would cause a console error `Invalid path` and it would not insert the reference. A work around is to click in the highlight again, and try to insert the reference a second time and that usually works. Not sure how to reproduce, but probably also not related to the changes from this PR.

### CODE REVIEW

**Review by lagartoverde — APPROVED** [review]
Looks really good, good job, such a big change :)
The sidebar maybe looks a little bit bigger than before (but maybe it's just my computer, so I would wait for a review from Dieter before merging, but it's a approval from my side ^^

**Comment by Dietr** [comment]
Did some design tweaks and used additional appuniversum classes and components where possible.

**Review by Dietr — APPROVED** [review]

**Pull request merged**
